app/src/main/res/layout/view_contributor_cell.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        >
+
+    <data>
+
+        <variable
+                name="viewModel"
+                type="io.github.droidkaigi.confsched2017.viewmodel.ContributorViewModel"
+                />
+    </data>
+
+    <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="?attr/selectableItemBackground"
+            android:clickable="true"
+            android:onClick="@{viewModel::onClickContributor}"
+            >
+
+        <ImageView
+                android:id="@+id/avatar"
+                android:layout_width="@dimen/contributor_image"
+                android:layout_height="@dimen/contributor_image"
+                android:layout_centerHorizontal="true"
+                android:layout_margin="@dimen/contributor_cell_margin"
```

Comment:
@konifar Do you mean to change only ImageView's margin bottom to 8dp while keeping margin top, left, right 16dp?
I also have a TextView under the ImageView with its margin bottom 16dp. What do you think of this margin bottom?

Just FYI, below is *all* margins are set to 8dp including ImageView's margins and TextView's margin bottom.
![contributors_design_fix](https://cloud.githubusercontent.com/assets/1192694/22674791/3eb848f2-ed25-11e6-863a-84a76db7a0ed.png)